.github/workflows/handle_llvm_runner_image.yml

```diff
@@ -9,53 +9,22 @@ on:
       - main
 
 jobs:
-  check_changes:
-    runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      pull-requests: read
-    outputs:
-      image_to_build: ${{ steps.set_output.outputs.image_to_build }}
-    steps:
-      - uses: actions/checkout@v4
-
-      - id: determine_changes
-        uses: tj-actions/changed-files@v41
-        with:
-          files_yaml: |
-            llvm_runner:
-              - "images/llvm_runner/Dockerfile"
-            llvm_runner_jammy:
-              - "images/llvm_runner/jammy.Dockerfile"
-            both:
-              - ".github/workflows/handle_llvm_runner_image.yml"
-
-      - id: set_output
-        run: |
-          if [ "${{ steps.determine_changes.outputs.llvm_runner_any_changed }}" == "true" ]; then
-            echo "image_to_build=llvm_runner" >> "$GITHUB_OUTPUT"
-          elif [ "${{ steps.determine_changes.outputs.llvm_runner_jammy_any_changed }}" == "true" ]; then
-            echo "image_to_build=llvm_runner_jammy" >> "$GITHUB_OUTPUT"
-          elif [ "${{ steps.determine_changes.outputs.both_any_changed }}" == "true" ]; then
-            echo "image_to_build=both" >> "$GITHUB_OUTPUT"
-          else
-            echo "image_to_build=none" >> "$GITHUB_OUTPUT"
-          fi
-
   handle_image:
-    needs: check_changes
-    if: needs.check_changes.outputs.image_to_build != 'none'
-    runs-on: [matterlabs-ci-runner]
+    runs-on: matterlabs-ci-runner
     strategy:
       matrix:
```

Comment:
Keep the matrix job to be able to add new images later on without major CI changes. `tag_prefix` equals to image name to simplify the process.